### PR TITLE
Us15 widget curiosity

### PR DESCRIPTION
**Pull request by ClemDSC**
final update

### CODE REVIEW

File: frontend/src/components/Curiosity/CuriosityPics.jsx [hunk 1]
```diff
@@ -42,59 +49,105 @@ export default function CuriosityPics({ picList }) {
...
+          </option>
+          <option value="MARDI">
+            Mars Descent Imager - MARDI -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("MARDI")).length}{" "}
+            photos
+          </option>
+          <option value="NAVCAM">
+            Navigation Camera - NAVCAM -{" "}
+            {picList.filter((pic) => pic.camera.name.includes("NAVCAM")).length}{" "}
+            photos
```
Comment (lchastanet) [inline frontend/src/components/Curiosity/CuriosityPics.jsx:89]: Too many repetitions, that could be improved !

**Review by lchastanet — CHANGES_REQUESTED** [review]

**Review by Lora048 — APPROVED** [review]
Tout est ok :)

File: frontend/src/components/Curiosity/CuriosityPics.jsx (hunk 1 above)
Comment (ClemDSC) [reply frontend/src/components/Curiosity/CuriosityPics.jsx]: New commit done to fix that

**Review by lchastanet — APPROVED** [review]
Nickel 👌🏻

**Pull request merged**